contracts/base/HooksManager.sol

```diff
@@ -13,9 +14,6 @@ contract HooksManager {
     // Events
     event HooksChanged(address indexed safe, address indexed hooksAddress);
 
-    // Errors
-    error AddressDoesNotImplementHooksInterface(address hooksAddress);
-
     /**
      * @notice Returns the address of hooks for a Safe account provided as a fucntion parameter.
      *         Returns address(0) is no hooks are enabled.
```

Comment:
I'd add a @dev notice about why the check is omitted in the NatSpec

Reply:
We do have the check now.